content/courses/onchain-development/anchor-pdas.md

```diff
@@ -310,42 +312,46 @@ pub struct ReallocExample<'info> {
 }
 
 #[account]
+#[derive(InitSpace)]
 pub struct AccountType {
     pub data: String,
 }
+
+const DISCRIMINATOR: u8 = 8;
```

Comment:
Why not just use `usize` for the DISCRIMINATOR and then not have to do the `as` conversion above?